pmm/pmm2-server-release.groovy

```diff
@@ -171,24 +171,28 @@ pipeline {
                     DOCKER_MID="\$TOP_VER.\$MID_VER"
                     sg docker -c "
                         set -ex
+                        # push pmm-server
                         docker pull \${DOCKER_VERSION}
+                        docker tag \${DOCKER_VERSION} percona/pmm-server:latest
+                        docker push percona/pmm-server:latest
+                        docker tag \${DOCKER_VERSION} perconalab/pmm-server:latest
+                        docker push perconalab/pmm-server:latest
+
                         docker tag \${DOCKER_VERSION} percona/pmm-server:\${VERSION}
                         docker tag \${DOCKER_VERSION} percona/pmm-server:\${DOCKER_MID}
                         docker tag \${DOCKER_VERSION} percona/pmm-server:\${TOP_VER}
                         docker push percona/pmm-server:\${VERSION}
                         docker push percona/pmm-server:\${DOCKER_MID}
                         docker push percona/pmm-server:\${TOP_VER}
```

Comment:
For the client below, we also push those version to perconalab. Probably we should do the same there for consistency

Reply:
Done!